src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -140,11 +157,32 @@ public bool ExecuteCommand(FormatDocumentCommandArgs args, CommandExecutionConte
                 }
 
                 progressTracker.ItemCompleted();
-                return;
+            }
+            else
+            {
+                Format(args.TextView, document, selectionOpt: null, cancellationToken);
+            }
+        }
+
+        private static bool TurnOnCodeCleanupForGroupBIfInABTest(Document document, Workspace workspace)
+        {
+            // If the feature is OFF and the feature options have not yet been Enabled to their assigned flight, do so now
+            // Do not reset the options if we already set it for them once, so options won't get reset if user manually disabled it already after it is enabled by the flight
+            if (!workspace.Options.GetOption(CodeCleanupOptions.PerformAdditionalCodeCleanupDuringFormatting, document.Project.Language)
+                && !workspace.Options.GetOption(CodeCleanupABTestOptions.SettingIsAlreadyUpdatedByExperiment))
+            {
+                var experimentationService = document.Project.Solution.Workspace.Services.GetService<IExperimentationService>();
+
+                if (experimentationService != null
+                    && experimentationService.IsExperimentEnabled(s_experimentName))
+                {
+                    workspace.Options = workspace.Options.WithChangedOption(CodeCleanupABTestOptions.SettingIsAlreadyUpdatedByExperiment, true)
+                                                         .WithChangedOption(CodeCleanupOptions.PerformAdditionalCodeCleanupDuringFormatting, document.Project.Language, true);
```

Comment:
didn't we say we shouldn't set PerformAdditionalCodeCleanupDuringFormatting?

Reply:
just talked with @jinujoseph so this is by design. we want to intentionally make it stick.